testing/compile.ts

```diff
@@ -271,6 +273,12 @@ export async function compilePackBundle({
     throw await processVmError(err, tempBundlePath);
   }
 
+  // Write the generated metadata. It's not used by the upload command, but
+  // it's helpful for debugging upload validation errors.
+  const metadata = await importManifest<PackVersionDefinition>(tempBundlePath);
+  const tempMetadataPath = path.join(intermediateOutputDirectory, 'metadata.json');
+  fs.writeFileSync(tempMetadataPath, JSON.stringify(metadata));
```

Comment:
This should be writing the result of `compilePackMetadata`, right?

Reply:
Good catch, fixed